src/java-opentimelineio/src/main/java/io/opentimeline/opentimelineio/Any.java

```diff
@@ -0,0 +1,169 @@
+package io.opentimeline.opentimelineio;
+
+import io.opentimeline.OTIONative;
+import io.opentimeline.opentime.RationalTime;
+import io.opentimeline.opentime.TimeRange;
+import io.opentimeline.opentime.TimeTransform;
+
+public class Any extends OTIONative {
+
+    public Any(long nativeHandle) {
+        this.nativeHandle = nativeHandle;
+    }
+
+    public Any(boolean b) {
+        this.initBool(b);
+    }
+
+    public Any(int i) {
+        this.initInt(i);
+    }
+
+    public Any(double d) {
+        this.initDouble(d);
+    }
+
+    public Any(String string) {
+        this.initString(string);
+    }
+
+    public Any(RationalTime rationalTime) {
+        this.initRationalTime(rationalTime);
+    }
+
+    public Any(TimeRange timeRange) {
+        this.initTimeRange(timeRange);
+    }
+
+    public Any(TimeTransform timeTransform) {
+        this.initTimeTransform(timeTransform);
+    }
+
+    public Any(AnyDictionary anyDictionary) {
+        this.initAnyDictionary(anyDictionary);
+    }
+
+    public Any(AnyVector anyVector) {
+        this.initAnyVector(anyVector);
+    }
+
+    public Any(SerializableObject serializableObject) {
+        this.initSerializableObject(serializableObject);
+    }
+
+    private void initBool(boolean b) {
+        this.className = this.getClass().getCanonicalName();
+        this.initializeBool(b);
+    }
+
+    private void initInt(int i) {
+        this.className = this.getClass().getCanonicalName();
+        this.initializeInt(i);
+    }
+
+//    private void initLong(long a) {
```

Comment:
was this commented out on purpose?

Reply:
Yes. The constructor with the `long` argument is for the native handle. So how do I initialise an Any with long? I could use a create any method, create a long Any on the native side and separately set the native handle. Someone could mistake the long constructor for a long Any initialisation constructor. 😅

Reply:
True, would anyone need to initialize Any with handle rather then long payload?
I would try to separate the payload from the reference, to avoid the confusion.

Reply:
That constructor would come in handy when we need to return an Any from the native side. If you have a pointer to a cpp any, we could create a java object on the cpp side using this long constructor. 

This is used in AnyDictionary when we need to get the value for a key. Even if we scrap the current AnyDictionary wrapper we would still require the method because we would need to create a Java Map on the native side from a cpp map.

Reply:
The nativeHandle constructors in all of the java bindings are for the same use case. The end user will never need to use it.

Reply:
Understood, maybe there is a way to have a constructor with something similar to `OTIONative` or like object for the C++ side of the initiation. It favors composition over inheritance, for example of `OTIONative` was part of Any, you can always initialize it for normal types or pass it in, when you already have the handle. Let me know if that makes sense.

Reply:
Do you mean to say that instead of all the classes inheriting from OTIONative they hold an OTIONative object? 

This will require some changes but seems viable.

Reply:
Correct, you can maybe even make the constructor with OTIONative package private so that it's only accessible from C++ but not from user's package.